wren/client/ent_client.go

```diff
@@ -67,13 +66,11 @@ func NewEntConfig(dbDialect string, dbName *string, dbHost *string, inMemory *bo
 		log.Fatal().Msg("Flag: `db-dialect` must be set to either `postgres` or `sqlite`.")
 	}
 
-	if dbPath != nil && strings.HasPrefix(*dbPath, "~/") {
-		usr, err := user.Current()
+	if dbPath != nil {
+		path, err := filepath.Abs(*dbPath)
```

Comment:
I'm not sure this will fix the problem -- for me, the directory was `tests/integration/data`, but only `tests/integration` existed. `filepath.Abs` doesn't seem to do anything about creating the directory.

Or is the problem that since the default path didn't start with `~/`, the `strings.HasPrefix(...)` above caused us not to run the normalization logic (the `MkdirAll` below)?

Reply:
If it's the latter -- this looks good but update the PR description to explain the actual fix (eg., always creating the directory fixes the problem, not calling `Abs`).

Reply:
Its the latter. I'll update the PR. Thanks for taking a look!